tox.ini

```
@@ -35,7 +35,7 @@ deps =
 
 commands =
     test: coverage erase
-    test: coverage run -m pytest {posargs:pydoctor}
+    test: coverage run -m pytest {posargs:pydoctor} -vv
```

Maybe add is as the default...but make it possible to disable it.

```suggestion
    test: coverage run -m pytest {posargs:-vv pydoctor}
```